tensorflow_model_optimization/python/core/clustering/keras/cluster_test.py

```diff
@@ -385,6 +385,48 @@ def testClusterFunctionalModelPreservesBuiltState(self):
           json.loads(clustered_model.to_json()))
     self.assertEqual(loaded_model.built, True)
 
+  @keras_parameterized.run_all_keras_modes
```

Comment:
it looks that the block of these tests  has not been upstreamed. i can remove it from my PR as they are not relevant